### PR TITLE
Replace policy with extension at WG page.

### PR DESCRIPTION
**Pull request by bianpengyuan**

### CODE REVIEW

**Comment by bianpengyuan** [comment]
cc @douglas-reid @mandarjog

**Review by douglas-reid — APPROVED** [review]

File: WORKING-GROUPS.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The current working groups are:
...
-| Policies and Telemetry | [Folder](https://drive.google.com/corp/drive/u/0/folders/0BzW5bSyKst8JbWtHOEo1STc1dGM) | [Forum](https://discuss.istio.io/c/policies-and-telemetry) | [#policies-telemetry](https://istio.slack.com/messages/C382V8Q92/) | [Notes](https://docs.google.com/document/d/1pn9QdRcoyT_nxOwzklsiYpt7OQraaSDfmtN14XTOrN0/edit) | [Hangouts Meet](https://meet.google.com/cmo-jgug-uhj) | [YouTube](https://www.youtube.com/playlist?list=PL7wB27eZmdffF-9nyaw01Ni_0GOWBzaF4) | Istio extensions for features such as Rate Limiting, Tracing, Monitoring, Logging
+| Extensions and Telemetry | [Folder](https://drive.google.com/corp/drive/u/0/folders/0BzW5bSyKst8JbWtHOEo1STc1dGM) | [Forum](https://discuss.istio.io/c/policies-and-telemetry) | [#extensions-telemetry](https://istio.slack.com/messages/C382V8Q92/) | [Notes](https://docs.google.com/document/d/1pn9QdRcoyT_nxOwzklsiYpt7OQraaSDfmtN14XTOrN0/edit) | [Hangouts Meet](https://meet.google.com/cmo-jgug-uhj) | [YouTube](https://www.youtube.com/playlist?list=PL7wB27eZmdffF-9nyaw01Ni_0GOWBzaF4) | Istio extensions for features such as Rate Limiting, Tracing, Monitoring, Logging
```
Comment (craigbox) [inline WORKING-GROUPS.md:28]: and Wasm too, or do you have separate meetings for that?

Comment (bianpengyuan) [reply WORKING-GROUPS.md]: Added Wasm related information. We do have a separated meeting for Wasm SIG, I listed it here as well.

**Comment by bianpengyuan** [comment]
ptal thanks!

**Review by nrjpoddar — APPROVED** [review]

**Pull request merged**